Unit3/Model/Config/Options.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace RLTSquare\Unit3\Model\Config;
+
+use Magento\Framework\Data\OptionSourceInterface;
+
+class Options implements OptionSourceInterface
+{
+    public function toOptionArray()
+    {
+
+        $options = [];
+        $options[] = [
+            'value' => 'Staging',
```

Comment:
You should declare value keys here as constants, e.g. "const ENV_STAGING = 'staging'", so you can later use RLTSquare\UnitThree\Model\Config\Source\Select::ENV_STAGING in if statements and other comparisons you need.